performance/Makefile

```diff
@@ -0,0 +1,56 @@
+# Docker image
+IMAGE_REPO ?= determinedai/perf-test
+IMAGE_TAG ?= $(USER)
+
+# Perf testing cluster
+CLUSTER_ID ?= $(USER)-perf
+KEYPAIR_ID ?= $(USER)
```

Comment:
idk if this is typically true, so maybe add a README to explain how to override it.

Reply:
It's not meant to be necessarily true, but more so something that'll break intentionally -- we don't have any real mandate on key pair names for individual devs, so not much to do here with it.
I can add an initial README that lists the overridable Makefile vars for now

Reply:
I think a good readme for this and how to config your make calls will probably save time in the long run